widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -444,6 +431,40 @@ public void startVisitorVideoView(MediaState visitorMediaState) {
                 .getScreenSharingController(screenSharingCallback);
     }
 
+    private void shouldShowVisitorVideo(CallState state) {
+        if (
```

Comment:
Could this, and the operator video if, be changed in a way that for example the `Media.Status.PLAYING` is contained into a variable? So that it reads "if state is 2 way video call and isVisitorVideoStatusPlaying and visibility is gone, do this". cause right now that PLAYING part is not the easiest to read.

Reply:
extracted to state